test/instrumentation/modules/@elastic/elasticsearch.test.js

```diff
@@ -348,11 +392,16 @@ if (semver.gte(pkgVersion, '7.7.0')) {
 
         const err = data.errors
           .filter((e) => e.exception.type === 'RequestAbortedError')[0]
-        t.ok(err, 'sent an error to APM server')
-        t.ok(err.id, 'err.id')
-        t.equal(err.exception.message, 'Request aborted', 'err.exception.message')
-        t.equal(err.exception.type, 'RequestAbortedError',
-          'err.exception.type is RequestAbortedError')
+        if (semver.eq(esVersion, '7.14.0')) {
```

Comment:
Presuming this is intentional, but just calling out that this is an `eq` test, which means if this behavior remains in 7.14.1, or 7.15.0 we'll have failures. No changes needed/expected.

Reply:
Yes, that was intentional... but perhaps just making more work for us. I'm slightly banking on the clients team getting a 7.14.1 or 7.15.0 that fixes https://github.com/elastic/elasticsearch-js/issues/1517